code/modules/mob/living/carbon/brain/MMI.dm

```diff
@@ -49,6 +49,11 @@
 			icon = 'icons/mob/alien.dmi'
 			icon_state = "AlienMMI"
 			alien = 1
+		else if(istype(O,/obj/item/organ/internal/brain/drask))

```

Comment:
How should it be handled then? Should I refactor how the bit above does it while I'm here?